tests/MongoDB.Driver.Core.TestHelpers/EventCapturer.cs

```diff
@@ -244,5 +254,40 @@ private BsonJavaScriptWithScope RecursivelyMaterialize(BsonJavaScriptWithScope v
                 return new BsonJavaScriptWithScope(value.Code, RecursivelyMaterialize(value.Scope));
             }
         }
+
+        private class EventNotifier
```

Comment:
Probably this class looks better as a nested rather than a separate as it was implemented in the previous commit.